res/values/strings_pl.arb

```diff
@@ -311,23 +311,23 @@
   "widgets_restore_from_blockheight" : "Przywróć z wysokości bloku",
   "widgets_restore_from_date" : "Przywróć od daty",
   "widgets_or" : "lub",
-  "widgets_seed" : "Ziarno",
+  "widgets_seed" : "Seed",
 
 
   "router_no_route" : "Brak zdefiniowanej trasy dla ${name}",
 
 
   "error_text_account_name" : "Nazwa konta może zawierać tylko litery, cyfry\ni musi mieć od 1 do 15 znaków",
-  "error_text_contact_name" : "Nazwa kontaktu nie może zawierać` , ' \" symbolika\ni musi mieć od 1 do 32 znaków ",
-  "error_text_address" : "Wallet address must correspond to the type\nof cryptocurrency",
+  "error_text_contact_name" : "Nazwa kontaktu nie może zawierać symboli ` , ' \"\ni musi mieć od 1 do 32 znaków ",
+  "error_text_address" : "Adres musi odpowiadać typowi kryptowaluty",
   "error_text_node_address" : "Wpisz adres iPv4",
   "error_text_node_port" : "Port węzła może zawierać tylko liczby od 0 do 65535",
   "error_text_payment_id" : "ID może zawierać od 16 do 64 znaków w formacie szesnastkowym",
   "error_text_xmr" : "Wartość XMR nie może przekraczać dostępnego salda.\nLiczba cyfr ułamkowych musi być mniejsza lub równa 12",
   "error_text_fiat" : "Wartość kwoty nie może przekroczyć dostępnego salda.\nLiczba cyfr ułamkowych musi być mniejsza lub równa 2",
-  "error_text_subaddress_name" : "Nazwa podadresu nie może zawierać ` , ' \" symbolika\ni musi mieć od 1 do 20 znaków",
+  "error_text_subaddress_name" : "Nazwa podadresu nie może zawierać symboli ` , ' \"\ni musi mieć od 1 do 20 znaków",
   "error_text_amount" : "Kwota może zawierać tylko liczby",
-  "error_text_wallet_name" : "Nazwa portfela może zawierać tylko litery, cyfry, _ - symbole\ni musi mieć od 1 do 33 znaków",
+  "error_text_wallet_name" : "Nazwa portfela może zawierać tylko litery, cyfry lub symbole _ - \ni musi mieć od 1 do 33 znaków",
```

Comment:
@OmarHatem28 Please confirm formatting on these when you get a chance

Reply:
yeah it's correct